### PR TITLE
feat(upload): adjust gif support

### PR DESCRIPTION
**Pull request by gary02**
resolve https://github.com/thematters/developer-resource/issues/253

### CODE REVIEW

File: src/components/Cover/index.tsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ export const Cover: React.FC<React.PropsWithChildren<CoverProps>> = ({
         url={url}
         width={1376}
         disabled={isFallback || inEditor}
+        disableAnimation={true}
```
Comment (robertu7) [inline src/components/Cover/index.tsx:26]: this component is used as user profile cover, tag profile cover, etc. and it's supposed to support animated GIF.

File: src/components/TagDigest/Feed/index.tsx [hunk 2]
```diff
@@ -126,6 +126,7 @@ const Feed = ({ tag, ...cardProps }: TagDigestFeedProps) => {
                   url={tag.cover || IMAGE_TAG_COVER.src}
                   width={144}
                   height={144}
+                  disableAnimation={true}
```
Comment (robertu7) [inline src/components/TagDigest/Feed/index.tsx:129]: please confirm if tag cover support animated GIF in feed.

File: src/components/TagDigest/Sidebar/index.tsx [hunk 3]
```diff
@@ -65,6 +65,7 @@ const Sidebar = ({ tag, ...cardProps }: TagDigestSidebarProps) => {
                 url={tag.cover || IMAGE_TAG_COVER.src}
                 width={144}
                 height={144}
+                disableAnimation={true}
```
Comment (robertu7) [inline src/components/TagDigest/Sidebar/index.tsx:68]: please confirm if tag cover support animated GIF in sidebar.

**Review by robertu7 — CHANGES_REQUESTED** [review]

File: src/common/utils/url.ts [hunk 4]
```diff
@@ -76,10 +78,14 @@ export const toSizedImageURL = ({
   const hostnameless = url.replace(urlDomain, ``)
   const key = hostnameless.replace('/public', '')
   const extedUrl = changeExt({ key, ext })
-  const postfix = height
+  let postfix = height
     ? `w=${width},h=${height},fit=crop`
     : `w=${width},h=${width * 4},fit=scale-down`
 
+  if (disableAnimation) {
```
Comment (robertu7) [inline src/common/utils/url.ts:85]: can disable by default instead since most of the use cases don't support animated GIF?

**Review by robertu7 — CHANGES_REQUESTED** [review]

File: src/common/utils/url.ts (hunk 4 above)
Comment (gary02) [reply src/common/utils/url.ts:85]: I think animation enable by default is a little more safe then disable by default ?

**Review by robertu7 — APPROVED** [review]

**Pull request merged**